### ISSUE TITLE
a-z Keystroke To Account Position in Menu

### ISSUE THREAD

**Issue by dougbutner** [greymass/anchor#1280]
### Platform

Desktop (MacOS)

### Description

I am requesting a feature for the account name selector where when a user presses a key that is a letter, the menu jumps to account starting with that letter. This is similar to the default display many browsers have for drop-down form elements.

This request is selfish - I could really use it - but also altruistic as I'm sure others would appreciate it. 

### Contact Details

dougbutner@gmail.com

**Comment by aaroncox** [comment]
Good call. I think I know exactly where you're asking for it to be, and I think it'd help me out in my day to day usage too.
